docs/protocol/mint_packages_nfts.md

```diff
@@ -4,23 +4,29 @@ This guide describes how to use the [Autonolas Protocol web app](https://protoco
 
 To mint a package (also referred to as _unit_) in the protocol, it must have been published into a remote IPFS registry. Read [how to publish packages](https://docs.autonolas.network/open-autonomy/guides/publish_fetch_packages/) with the {{open_autonomy}} framework.
 
-Minting a package involves filling in some data and creating a **metadata file**, which contains a pointer to the IPFS hash of the published software package.
-The [Autonolas Protocol web app](https://protocol.autonolas.network/) will automatically create and publish the metadata file on the IPFS registry, based on the package/unit information and the information provided by the user in the online form. Then, it will interact with the protocol smart contracts to complete minting the package on-chain.
+Minting a software package involves creating a **metadata file** that contains the package data and a pointer to the IPFS where the package is published.
+The [Autonolas Protocol web app](https://protocol.autonolas.network/) will automatically create and publish the metadata file on the IPFS registry, based on the package/unit information provided by the user in an online form. Then, it will interact with the protocol smart contracts to complete minting the package on-chain.
 
 Upon completing the minting process, you will have an NFT representing your package in the Autonolas Protocol. See the figure below.
 
 <figure markdown>
 ![Package minting](./images/package_minting.svg)
 </figure>
 
+Minted NFTs offers an on-chain representation of the relationship between different software packages by mirroring their dependency relations. This helps the protocol [Tokenomics](./tokenomics.md) to compute appropriate incentives, for example, based on how often a component is referenced.
+
+<figure markdown>
+![Package dependencies represented on-chain](./images/nft_dependencies.svg)
+</figure>
+
 ## Requirements
 
 In order to mint a software package as an NFT, you must ensure that you have:
 
 * An **address** associated to either
 
-    * a crypto wallet (e.g., [Metamask](https://metamask.io/) or a cold wallet), or
-    * a multisig contract (like [Safe](https://safe.global/)) which allows to connect via [Wallet Connect](https://walletconnect.com/).
+  * a crypto wallet (e.g., [Metamask](https://metamask.io/) or a cold wallet), or
```

Comment:
```suggestion
  * a crypto wallet (e.g., [Metamask](https://metamask.io/) or
```

Reply:
The dichotomy Metamask or cold wallet seems confusing, e.g. a cold wallet can be used with Metamask, effectively being Metamask in both cases what the user interacts with